samples/arcgis-widget/src/components/Daylight/Daylight.tsx

```diff
@@ -49,16 +54,24 @@ const Daylight = (
                 onWidgetDestroyed={onWidgetDestroyed}
             />
             {widget && (
-                <Select
-                    className="Daylight-select"
-                    onChange={(event) => {
-                        props.model.dateOrSeason = event.target.value as any;
-                    }}
-                    value={props.model.dateOrSeason}
-                >
-                    <MenuItem value="date">Date</MenuItem>
-                    <MenuItem value="season">Season</MenuItem>
-                </Select>
+                <>
```

Comment:
Is this fragment needed? There is only one root component.

Reply:
Negative, not needed. It was an artifact from when I had multiple elements at one point when refactoring.